src/libPMacc/include/pluginSystem/PluginConnector.hpp

```diff
@@ -180,6 +181,19 @@ namespace PMacc
             }
         }
 
+        template<typename Plugin>
+        Plugin* getPluginFromType()
```

Comment:
Please add a description to this function.
What is if there are more than one plugin with the same type inside the list named `plugins`?